public/index.html

```diff
@@ -7,8 +7,8 @@
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico" />
     <link
       rel="stylesheet"
-      href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css"
-      integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u"
+      href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css"
```

Comment:
These css changes are drastic, and the components look terrible inside of the updated css.  Please run "yarn start" to start up the demo locally (and compare to https://sage-bionetworks.github.io/Synapse-React-Client/ ).  The modal width seems to also be reduced (causing scroll bars and content issues), as well as using rem instead of px for all fonts (generally a great idea, but we're not ready to promote using 4.5.2 css until we make a lot of overrides).